tests/test_utils.py

```diff
@@ -11,24 +22,85 @@ class TestUtils:
     with open("tests/files/foo.webm", "rb") as fp:
         body_webm = fp.read()
 
-    @pytest.mark.parametrize("input_bytes,expected",[
-        (body_mp4,True),
-        (b"\x00\x00\x00",False),
-        (b"\x00\x00\x00 ftypmp4",False),
-        (b"\x00\x00\x00 ftypmp42",False),
-        (b"\x00\x00\x00 testmp42\x00\x00\x00\x00mp42mp41isomavc1",False),
-        (b"\x00\x00\x00 ftyp2222\x00\x00\x00\x002222mp41isomavc1",True),
-        (b"\x00\x00\x00 ftyp2222\x00\x00\x00\x0022222221isomavc1",False),
-        ])
+    with open("tests/files/NonID3.mp3", "rb") as fp:
+        body_mp3 = fp.read()
+
+    @pytest.mark.parametrize(
+        "input_bytes,expected",
+        [
+            (body_mp4, True),
+            (b"\x00\x00\x00", False),
+            (b"\x00\x00\x00 ftypmp4", False),
+            (b"\x00\x00\x00 ftypmp42", False),
+            (b"\x00\x00\x00 testmp42\x00\x00\x00\x00mp42mp41isomavc1", False),
+            (b"\x00\x00\x00 ftyp2222\x00\x00\x00\x002222mp41isomavc1", True),
+            (b"\x00\x00\x00 ftyp2222\x00\x00\x00\x0022222221isomavc1", False),
+        ],
+    )
     def test_is_mp4_signature(self, input_bytes, expected):
         assert _is_mp4_signature(input_bytes) == expected
 
-    @pytest.mark.parametrize("input_bytes,expected",[
-        (body_webm, True),
-        (b"\x00\x00\x00",False),
-        (b"\x1aF\xdf\xa3",False),
-        (b"\x1aE\xdf\xa3B\x82",False),
-        (b"\x1aE\xdf\xa3B\x82\x00\x00\x00",False),
-        ])
+    @pytest.mark.parametrize(
+        "input_bytes,expected",
+        [
+            (body_webm, True),
+            (b"\x00\x00\x00", False),
+            (b"\x1aF\xdf\xa3", False),
+            (b"\x1aE\xdf\xa3B\x82", False),
+            (b"\x1aE\xdf\xa3B\x82\x00\x00\x00", False),
+        ],
+    )
     def test_is_webm_signature(self, input_bytes, expected):
-        assert _is_webm_signature(input_bytes) == expected
\ No newline at end of file
+        assert _is_webm_signature(input_bytes) == expected
+
+    def test_parse_vint(self):
+        assert _parse_vint(self.body_webm, len(self.body_webm), 6) == 8
+        assert _parse_vint(self.body_webm, len(self.body_webm), 30) == 1
+
+    @pytest.mark.parametrize(
+        "input_bytes,expected",
+        [
+            (body_mp3, True),
+            (b"\x00\x00\x00", False),
+            (b"\xff\xfb\x90d\x00", False),
+            (body_mp3[:50], False),
+        ],
+    )
+    @mock.patch("xtractmime.utils._mp3_framesize")
+    def test_is_mp3_non_ID3_signature(self, mock_framesize, input_bytes, expected):
+        if input_bytes == self.body_mp3[:50]:
+            mock_framesize.return_value = 10
+            assert _is_mp3_non_ID3_signature(input_bytes) == expected
+        else:
+            mock_framesize.return_value = 417
+            assert _is_mp3_non_ID3_signature(input_bytes) == expected
```

Comment:
I suggest moving the value of `mock_framesize.return_value` to `@pytest.mark.parametrize`, even if it means repeating it 3 times.